cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -435,6 +435,82 @@ public void testNewToProposalReadyToRebalancingToReadyRebalance(VertxTestContext
             }));
     }
 
+    /**
+     * Tests the transition from 'New' to to 'ProposalReady'
+     * The rebalance proposal is approved and the resource moves to 'Rebalancing' and finally to 'Ready'
+     * Then the Ready KafkaRebalance is refreshed and a moved to 'ProposalReady' again.
+     *
+     * 1. A new KafkaRebalance resource is created; it is in the 'New' state
+     * 2. The operator requests a rebalance proposal through the Cruise Control REST API
+     * 3. The rebalance proposal is ready on the first call
+     * 4. The KafkaRebalance resource transitions to the 'ProposalReady' state
+     * 9. The KafkaRebalance resource is annotated with 'strimzi.io/rebalance=approve'
```

Comment:
Do these numbers refer to some state number or it is just leftover and it should be 4.....5. etc?